### DISCUSSION TITLE
Question about reacnetgenerator output file

### DISCUSSION THREAD

**Discussion by bit-wyj** [deepmodeling/reacnetgenerator#2236] [General]
Hello, I use the command line reacnetgenerator --type dump -i 3000k-1ps-pvdf.dump -a C H F --matrixa 0.91 0.09 0.09 0.91. The xx.species file can identify [H]F 1 and [H]C([H])C(F)C(F)(F)C([H])([H])C(F)(F)C([H])([H])[H] 1, but the xx.abcd file does not output the reaction of the reactant to form HF.![Uploading ff4e7e9583e876447aa8963e64217b3.png…]()
[dump5.zip](https://github.com/deepmodeling/reacnetgenerator/files/15495054/dump5.zip)

**Comment by njzjz** [comment]
This is a known issue when HMM is enabled. Results are reliable when HMM is disabled.

**Reply by bit-wyj** [reply]
[HF.zip](https://github.com/user-attachments/files/15524778/HF.zip)
You can check this HF.zip file. The lammpstrj file is scanned by Gaussian and is an obvious HF elimination reaction. However, the abcd file cannot directly output this HF elimination reaction. reacnetgenerator divides the reaction into two steps, first removing F atoms, and then F extracting H to generate HF. If users use the results of the abcd file, they will think that the initial reactant will remove a large number of F atoms instead of HF. How can this situation be avoided?
